blossom/templates/website/generic_form.html

```diff
@@ -136,9 +125,9 @@ <h4>{{ subheader }}</h4>
     </div>
 
     <script>
-        document.addEventListener("DOMContentLoaded", function (event) {
-            const inputs = document.querySelectorAll('.inputfile');
-            Array.prototype.forEach.call(inputs, function (input) {
+        document.addEventListener("DOMContentLoaded", function () {
+            const inputs = Array.from(document.getElementsByClassName('inputfile'));
```

Comment:
Since this is being chained into a `.forEach()`, why are we casting this as an Array? The NodeList object also contains a `.forEach()` implementation.

Reply:
I tried that first but it didn't seem to work:
https://stackoverflow.com/questions/49956141/how-to-iterate-on-htmlcollection

What would be better here is to use the `for .. of` syntax, but I'm not sure about browser support for that.

Reply:
Turns out this is possible for the query selectors, but not for the other function because it returns live elements and not a static list. I changed it back because the chained version is much cleaner.